src/clarity.rs

```diff
@@ -425,10 +425,11 @@ pub fn invoke_command(invoked_by: &str, args: &[String]) {
                     db.initialize();
                     db.begin();
                     for (principal, amount) in allocations.iter() {
-                        let cur_balance = db.get_account_stx_balance(principal);
-                        let final_balance = cur_balance.checked_add(*amount as u128).expect("FATAL: account balance overflow");
-                        db.set_account_stx_balance(principal, final_balance as u128);
-                        println!("{} credited: {} uSTX", principal, final_balance);
+                        let mut balance = STXBalance::zero();
```

Comment:
Should there be a `STXBalance::initial(x)`-like method that instantiates a balance with an initial value?  It seems weird that you're doing a separate `.credit()` operation, and having to pass in a magic constant `0` for the block height.

Reply:
Good idea

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1850/commits/a15d408578a925f9c77029f0480668fa87161e5a